build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/git_clone.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
why do we still need this git_clone.sh while we have the maxdome_post_startup.sh?

Reply:
Somehow I cannot see that file locally , I will delete it via the console